.circleci/config.yml

```diff
@@ -98,9 +98,105 @@ jobs:
           name: Testing
           command: make trace_test
           no_output_timeout: 1800s
+  coverage:
+    working_directory: ~/tikv-ci
```

Comment:
Can this be merged with test?

Reply:
We can. In my first attempt, I just added coverage in the test job.

But I found that the job takes a long time. I am concerning about the test time, if we just merge the two job, we can not utilize parallelization to reduce the test time.

Reply:
Currently most of the CI time is spend dealing with caches (discovered in #3075) so having more jobs likely will cause more slowdowns.

Reply:
Fine. I will merge the jobs.